launch_test.go

```diff
@@ -865,6 +867,32 @@ func TestFetchParentBuildMeta(t *testing.T) {
 	}
 }
 
+func TestFetchParentBuildsMetaParseError(t *testing.T) {
+	TestParentBuildIDs := []float64{1111, 2222}
+	IDs := make([]interface{}, len(TestParentBuildIDs))
```

Comment:
Why can't we just initialize `IDs` with the test values?

Reply:
I can't seem to initialize `IDs` in a simpler way. I think interface arrays are treated differently than int arrays.

Reply:
Fine as-is

Reply:
Moved the vars to a more global level